public/app/plugins/datasource/cloudwatch/components/ConfigEditor.tsx

```diff
@@ -229,21 +248,24 @@ export class ConfigEditor extends PureComponent<Props, State> {
               )}
             </div>
           )}
-          {options.jsonData.authType === 'arn' && (
-            <div className="gf-form-inline">
-              <div className="gf-form">
-                <InlineFormLabel className="width-14" tooltip="ARN of Assume Role">
-                  Assume Role ARN
-                </InlineFormLabel>
-                <div className="width-30">
-                  <Input
-                    className="width-30"
-                    placeholder="arn:aws:iam:*"
-                    value={options.jsonData.assumeRoleArn || ''}
-                    onChange={onUpdateDatasourceJsonDataOption(this.props, 'assumeRoleArn')}
-                  />
-                </div>
+          <div className="gf-form-inline">
+            <div className="gf-form">
+              <InlineFormLabel
+                className="width-14"
+                tooltip="ARN of role to assume. Specifying a role here will ensure that the selected authentication provider is used to assume the specified role rather than using the credentials directly. Leave blank if you don't need to assume a role at all"
```

Comment:
@achatterjee-grafana I don't think that's a good change, because it sounds as if you _should_ assume a role. It's optional, by default you don't assume one.

Reply:
Thanks for the explanation. How about "Optionally, specify a role to assume.".

Reply:
Done.